scripts/launch-xrd

```diff
@@ -243,12 +243,18 @@ while [[ $# -gt 0 ]]; do
             shift 2
             ;;
         -- )
-           shift
-           ;;
+            shift
+            if [[ ${#CTR_MGR_ARGS[@]} -ne 0 ]]; then
+                echo "When using --, use container manager args after it. These args are inavlid for the script ${CTR_MGR_ARGS[*]}" >&2
+                bad_usage
+            fi
```

Comment:
I'm not quite sure what this error message is meant for. You might want to check there's at least one arg after `--` (i.e. check `$#`), but that doesn't seem to be what you're doing here.